tests/testflows/aes_encryption/configs/clickhouse/config.xml

```diff
@@ -0,0 +1,436 @@
+<?xml version="1.0"?>
```

Comment:
Please do not copy/pase default config.xml. It makes impossible to enable features by default in all tests and prod. It is better to organize config as default one plus small differences. Such behavior is used in integration tests.

Reply:
I agree that this can be structured in a better way. However, this would affect everything in the `tests/testflows` folder. This  will be done in separate PR.